go/vt/tabletserver/planbuilder/dml.go

```diff
@@ -29,6 +29,11 @@ func analyzeUpdate(upd *sqlparser.Update, getTable TableGetter) (plan *ExecPlan,
 		return nil, err
 	}
 
+	// Store the WHERE clause as string for the transaction serializer.
+	buf := sqlparser.NewTrackedBuffer(nil)
+	buf.Myprintf("%v", upd.Where)
```

Comment:
See the method `computeTxSerializerKey()` in tabletserver.go.

I get the table name there. I think it's cleaner to store it separately. The txserializer package also uses it separately because the stat vars are only keyed by table name and not by table+WHERE to avoid too many entries in that map.